### PR TITLE
Add Doctest / Change Project import

### PR DESCRIPTION
**Pull request by takesei**
* TaguPy version:
* Python version:
* Operating System:

### What
- Add Doctest environment, compatible with old test env.
- Change file name, add `_` to every file to remove from python completion.
- You can now import core library directly (like import tagupy.generator)

### CODE REVIEW

File: tagupy/utils/_functions.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+
+def get_corr_matrix(exmatrix: np.ndarray, max_dim: int) -> np.ndarray:
+    """
+    Return Correlation Matrix
+
+    Parameters
+    ----------
+    max_dim: int
+        maximum dimension treated in alias matrix
```
Comment (takesei) [inline tagupy/utils/_functions.py:18]: x `alias matrix` → o `corr matrix`

**Review by takesei — COMMENTED** [review]
Fix typo

File: tagupy/utils/_functions.py (hunk 1 above)
Comment (takesei) [reply tagupy/utils/_functions.py]: なおした

File: .gitignore [hunk 2]
```diff
@@ -5,6 +5,7 @@
 __pycache__/
 *.py[cod]
 *$py.class
+.python-version
```
Comment (taka-sho) [inline .gitignore:8]: 👍🏻

**Review by taka-sho — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (takesei) [reply .gitignore:8]: 👍

**Review by yo-aka-gene — APPROVED** [review]
LGTM

**Comment by taka-sho** [comment]
LGTM

**Pull request merged**